### PR TITLE
[WIP] Display recent votes on homepage

### LINKED ISSUE #1101: Highlight recent votes on the homepage
A regular query from users is where they can find a recent vote. Generally they've read about it in the press, possibly with different wording than is used in the debate title (or they haven't thought/been able to search).

We could pull out recent votes and display them on the homepage. This would help the people above and also draw casual browsers' attention to what has actually been decided upon in Parliament recently. Less significantly, it would also save time for me, since I generally search on whatever small information they have given me - a date or subject matter - and, unless there's a trick I don't know, it's by no means easy to locate the debates where votes happened.

### PR DESCRIPTION
**Pull request by zarino**
Fixes #1101.

Used to look like: 

image

### CODE REVIEW

**Comment by zarino** [comment]
@dracos I’m running out of design time on this one before Christmas, so pinging you in case you have time to finish off?

Things on my mind:

1. Should recent votes _replace_ the "Random debate" we get if no featured debate has been chosen? Or maybe we _always_ show _a few_ recent votes, but increase the number if there’s no featured debate. @MyfanwyNixon might have thoughts.
2. I’m a bit worried that the list will get clogged up with aparent "duplicate" divisions – multiple divisions from the same debate, with identical—or very similar—titles. Perhaps we group by debate (and therefore show the _debate_ title rather than the division title, and link to the _debate_ page rather than the division page), to avoid that?

**Comment by MyfanwyNixon** [comment]
I don't feel that strongly, but I do quite like to random debate for its potential to pull people into content they might not otherwise have found. So I think I'd be in favour of keeping it and always having the votes separately.

File: www/docs/style/sass/pages/_home.scss [hunk 1]
```diff
@@ -327,10 +327,10 @@ input.homepage-search__button {
 
 .excerpt__title {
     margin-bottom: 0.1em;
-    max-width: 21em;
     font-size: 1.125em;
     @media (min-width: $medium-screen) {
         font-size: 1em;
+        padding-right: 10em;
```
Comment (dracos) [inline www/docs/style/sass/pages/_home.scss]: Not sure this was enough:
screen shot 2018-12-20 at 09 49 39

Comment (zarino) [reply www/docs/style/sass/pages/_home.scss]: Ah poop, didn’t spot "Written ministerial statements"!

**Comment by zarino** [comment]
@dracos I’ve pushed two commits that:

1. Move the recent votes down into the second homepage section, just before the "Recently in Parliament" heading.
2. Fix that overlapping `.excerpt__category` bug you spotted.

On point 1 – it was tempting put the list of votes _inside_ that "Recently in Parliament" list, since divisions _do_ happen in Parliament, but then you’d have an odd mix of votes and speeches under a single heading. Or you’d have a frankenheading, like the "Public bill committees" heading-thats-not-a-heading. Keeping them separate felt neater, but let me know if you disagree.

**Pull request merged**